### PR TITLE
Update busy on busy description for calling policy

### PR DESCRIPTION
**Pull request by xinawang**

### CODE REVIEW

File: Teams/teams-calling-policy.md [hunk 1]
```diff
@@ -101,12 +101,12 @@ This setting allows you to turn on or turn off music on hold when a PSTN caller
...
+Busy on busy when in a call (also called "busy options") lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. New or incoming calls can be rejected with a busy signal or can be routed accordingly to the user's unanswered settings. Regardless of how their busy options are configured, users in a call or conference or those with a call on hold are not prevented from initiating new calls or conferences. This setting is set to **Not enabled** by default.
 
 - **Not enabled** No busy option is enabled and new or incoming calls can still go to the user while the user is already in a call.
 - **Enabled** New or incoming calls will be rejected with a busy signal.
 - **Unanswered** The user's unanswered settings will be used, such as routing to voicemail or forwarding to another user.
-- **User controlled** This option isn't currently functional; if set, this will be read as setting the value to **Not enabled**.
+- **User controlled** Users can determine whether their busy options choice from Calls settings in Teams app.
```
Comment (JohanFreelancer9) [inline Teams/teams-calling-policy.md:109]:
```suggestion
- **User controlled** Users can determine their busy options choice from call settings in the Teams app. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xinawang, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by xinawang** [comment]
Anything else needed to merge the PR?

**Comment by JohanFreelancer9** [comment]
> Anything else needed to merge the PR?

@scanum

**Comment by xinawang** [comment]
What is the merge process? We are in the process of rolling out the feature so doc needs to updated. Thank you

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**